libraries/SuplaDevice/examples/Supla_MAX6675_No_SSL/Supla_MAX6675_No_SSL.ino

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (C) AC SOFTWARE SP. Z O.O.
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+*/
+
+#include <SPI.h>
+#include <SuplaDevice.h>
+
+
+// Add include to MAX6675_K sensor
+#include <supla/sensor/MAX6675_K.h>
+
```

Comment:
please add netowrk options for Arudino Mega ehternet shield and make sure it compiles on AVR board target.